src/engine/ruleset/decoders/sonicwall/firewall-general.yml

```diff
@@ -0,0 +1,63 @@
+---
+name: decoder/firewall-general/0
+
+metadata:
+  module: firewall
+  title: Firewall logs decoder
+  description: Decoder for Linux firewall logs
+  author:
+    name: Wazuh Inc. info@wazuh.com
+    date: 2023-01-12
+  references:
+    - https://www.sonicwall.com/es-mx/support/technical-documentation
+
+sources:
+  - decoder/queue-localfile/0
```

Comment:
The syslog queue should be added, since events can be received by the wazuh syslog server. 
I am not so sure that it is necessary to add the localfile queue.